spec/models/manageiq/providers/openshift/container_manager/container_template_spec.rb

```diff
@@ -0,0 +1,61 @@
+describe ContainerTemplate do
+  let(:ems) do
+    hostname = 'host.example.com'
+    token = 'theToken'
+    FactoryGirl.create(
+      :ems_openshift,
+      :name                      => 'OpenShiftProvider',
+      :connection_configurations => [{:endpoint       => {:role     => :default,
+                                                          :hostname => hostname,
+                                                          :port     => "8443"},
+                                      :authentication => {:role     => :bearer,
+                                                          :auth_key => token,
+                                                          :userid   => "_"}},
+                                     {:endpoint       => {:role     => :hawkular,
+                                                          :hostname => hostname,
+                                                          :port     => "443"},
+                                      :authentication => {:role     => :hawkular,
+                                                          :auth_key => token,
+                                                          :userid   => "_"}}]
```

Comment:
We actually have a factory for this provider. I think it might be more convenient this way in case you need to rerecord the changes (and change the hostname, password etc.)